libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.ts

```diff
@@ -0,0 +1,113 @@
+import { CommonModule } from '@angular/common';
+import {
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+  TemplateRef,
+  inject,
+} from '@angular/core';
+import { SkyIdModule, SkyIdService, SkyTrimModule } from '@skyux/core';
+import { SkyIconModule } from '@skyux/indicators';
+import { SkyPopoverModule } from '@skyux/popovers';
+import { SkyThemeModule } from '@skyux/theme';
+
+import { SkyHelpInlineResourcesModule } from '../shared/sky-help-inline-resources.module';
+
+import { SkyHelpInlineAriaExpandedPipe } from './help-inline-aria-expanded.pipe';
+
+/**
+ * @internal
+ */
+@Component({
+  selector: 'sky-help-inline',
+  standalone: true,
+  templateUrl: './help-inline.component.html',
+  styleUrls: ['./help-inline.component.scss'],
+  imports: [
+    CommonModule,
+    SkyIdModule,
+    SkyTrimModule,
+    SkyPopoverModule,
+    SkyIconModule,
+    SkyThemeModule,
+    SkyHelpInlineAriaExpandedPipe,
+    SkyHelpInlineResourcesModule,
+  ],
+})
+export class SkyHelpInlineComponent {
+  readonly #idSvc = inject(SkyIdService);
+
+  protected popoverId: string | undefined;
+  protected popoverTemplate: TemplateRef<unknown> | undefined;
+  protected popoverOpenedFlag: boolean | undefined;
```

Comment:
Let's name this `isPopoverOpened` to be consistent with convention: starting booleans with a "be" verb.